chrome/nav.js

```diff
@@ -1141,6 +1152,56 @@ function updateRoutePlanner( data ) {
 	}
 }
 
+function highlightVisited( data ) {
+	let ukey = Universe.getServer ( doc ).substr( 0, 1 );
+
+    // initial setup
+    if ( !data[ ukey + 'visit' ] ) {
+        data[ ukey + 'visit' ] = {};
+    }
+    if( !config.displayVisitedDecay ) {
```

Comment:
default case should match what's in the options imo,
suggest putting 180 as default number in the options
no zero for default value